src/main/java/ContactManager.java

```diff
@@ -63,35 +76,28 @@ boolean checkForContacts() {
         }
     }
 
-    void updateContactFields(Contact contact) {
+    public void updateContactFields(Contact contact) {
         System.out.println("Currently: " + contact.returnFirstName());
-        contact.updateFirstName(consoleInput.confirmInput("first name"));
+        contact.updateFirstName(consoleInput.confirmInput("first name", true));
```

Comment:
Try to remove the knowledge of "updating" from the console input and move it to somewhere that needs to know about it.